cinder/volume/drivers/vmware/datastore.py

```diff
@@ -66,7 +68,9 @@ def __init__(self, vops, session, max_objects, ds_regex=None,
         self._profile_id_cache = {}
         self._random_ds = random_ds
         self._random_ds_range = random_ds_range
+        self._host_props_cache = {}
 
+    @cached(cache=TTLCache(maxsize=64, ttl=300))
```

Comment:
This seems to have a built-in cache `self._profile_id_cache`. What's the point of adding another layer of caching?